gaming/README.md

```diff
@@ -0,0 +1,153 @@
+# Spanner Gaming Samples
+
+This repository contains sample code for the following use-cases when using Cloud Spanner for the backend:
+
+- Player creation, login, and skin changes
+
+## How to use this
+
+### Setup infrastructure
+A terraform file is provided that creates the appropriate resources for these samples.
+
+Resources that are created:
+- Spanner instance and database based on user variables in main.tfvars
+- (FUTURE) GKE cluster to run the load generators
+
+To set up the infrastructure, do the following:
+
+- Copy `infrastructure/terraform.tfvars.sample` to `infrastructure/terraform.tfvars`
+- Modify `infrastructure/terraform.tfvars` for PROJECT and instance configuration
+- `terraform apply` from within infrastructure directory
+
+```
+cd infrastructure
+cp terraform.tfvars.sample terraform.tfvars
+vi terraform.tfvars # modify variables
+
+terraform apply
+```
+
+### Setup schema
+Schema is managed by [Wrench](https://github.com/cloudspannerecosystem/wrench).
```

Comment:
The primary goal of sample apps is to demonstrate the usage of Spanner focusing on Schema and client library usage. While schema management may be a good topic to dive into, it seems to deviate from the core goals and is distracting. I'd recommend removing this from this PR. Please consider following up with a separate PR if necessary after thorough evaluation with team members.